packages/browser/src/login/oidc/IssuerConfigFetcher.ts

```diff
@@ -139,7 +139,9 @@ function processConfig(
       parsedConfig[issuerConfigKeyMap[key].toKey] = config[key];
     }
   });
-
+  if (parsedConfig.scopesSupported === undefined) {
```

Comment:
Perhaps we can be more defensive?

```suggestion
  if (!Array.isArray(parsedConfig.scopesSupported)) {
```